3-semantic-html/basic.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Basic semantic in HTML</title>
+    </head>
+    
+    <body>
+        <header>
+            <h1>The VIM editor</h1>
+            <nav>
+                <ul>
+                    <li>Menu</li>
+                    <li>Colors</li>
+                </ul>
+            </nav>
+        </header>
+
+        <main>
+            <article>
+                <header>
+                    <h2>Vim editor</h2>
+                </header>
+                <section>
+                    <p>
+                        Vim is a clon with additions of "VI" the editor made for Unix OS, this was released to public in 1991. Vim is designed for use both from a CLI(<mark>command line interface</mark>) as and a standalone application in a graphic user interface.
+                        <blockquote>
+                            Vim's interface is not based on menus or icons but on commands given in a text user interface.
+                        </blockquote>
+                        Vim has 12 modes, here i mention the basic ones:
+                    </p>
+                    <ol>
+                        <li>Normal</li>
+                        <li>Visual</li>
+                        <li>Select</li>
+                        <li>Insert</li>
+                        <li>Command Line</li>
+                        <li>Ex</li>
+                    </ol>
+                </section>
+                <footer>
+                    <p>
+                        Vim has customization, more features, scripts and much more but you can read about this in the web ...
+                        <br>
+                        References : <cite>Wikipedia</cite> <br>
+                        <address>
+                            <a href="https://en.wikipedia.org/wiki/Vim_(text_editor)" target="blank">Vim on wikipedia !</a>
+                        </address>
```

Comment:
```suggestion
                            <a href="https://en.wikipedia.org/wiki/Vim_(text_editor)" target="blank">Vim on wikipedia !</a>
```
`<address>` is more commonly used for physical addresses and contact information.  if you want to provide a url, accept the suggestion.